device/ios/Plugin/DevicePlugin.swift

```diff
@@ -24,6 +24,7 @@ public class DevicePlugin: CAPPlugin {
         let diskFree = implementation.getFreeDiskSize() ?? 0
         let realDiskFree = implementation.getRealFreeDiskSize() ?? 0
         let diskTotal = implementation.getTotalDiskSize() ?? 0
+        let modelName = isSimulator ? ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"]! + " Simulator" : implementation.getModelName()
```

Comment:
Don’t force unwrap. I thought we had a lint rule to prevent it, but looks like we don’t.
Also, now that we can get the actual name I don’t think the model should contain the “Simulator” part, if users need to know if it’s a simulator they can check the “isVirtual” property